Index: dev_appserver.py
===================================================================
--- dev_appserver.py	(revision 77)
+++ dev_appserver.py	(working copy)
@@ -37,6 +37,7 @@
 SCRIPT_DIR = os.path.join(DIR_PATH, 'google', 'appengine', 'tools')
 
 EXTRA_PATHS = [
+  "PATH/TO/MONGO-APPENGINE-CONNECTOR/DIRECTORY",
   DIR_PATH,
   os.path.join(DIR_PATH, 'lib', 'antlr3'),
   os.path.join(DIR_PATH, 'lib', 'django'),
Index: google/appengine/tools/dev_appserver.py
===================================================================
--- google/appengine/tools/dev_appserver.py	(revision 77)
+++ google/appengine/tools/dev_appserver.py	(working copy)
@@ -84,7 +84,7 @@
 from google.appengine.api import appinfo
 from google.appengine.api import croninfo
 from google.appengine.api import datastore_admin
-from google.appengine.api import datastore_file_stub
+import datastore_mongo_stub
 from google.appengine.api import mail_stub
 from google.appengine.api import urlfetch_stub
 from google.appengine.api import user_service_stub
@@ -3311,9 +3311,8 @@
 
   apiproxy_stub_map.apiproxy = apiproxy_stub_map.APIProxyStubMap()
 
-  datastore = datastore_file_stub.DatastoreFileStub(
-      app_id, datastore_path, history_path, require_indexes=require_indexes,
-      trusted=trusted)
+  datastore = datastore_mongo_stub.DatastoreMongoStub(
+      app_id, datastore_path, history_path, require_indexes=require_indexes)
   apiproxy_stub_map.apiproxy.RegisterStub('datastore_v3', datastore)
 
   fixed_login_url = '%s?%s=%%s' % (login_url,
